pkg/injector/pod_patch.go

```diff
@@ -271,6 +281,50 @@ func profilingEnabled(annotations map[string]string) bool {
 	}
 }
 
+func getResourceRequirements(annotations map[string]string) (*v1.ResourceRequirements, error) {
+	r := v1.ResourceRequirements{
+		Limits:   v1.ResourceList{},
+		Requests: v1.ResourceList{},
+	}
+	cpuLimit, ok := annotations[daprCPULimitKey]
+	if ok {
+		cpuLimit, err := resource.ParseQuantity(cpuLimit)
+		if err != nil {
+			return nil, fmt.Errorf("error parsing sidecar cpu limit: %s", err)
+		}
+		r.Limits[v1.ResourceCPU] = cpuLimit
+	}
+	memLimit, ok := annotations[daprMemoryLimitKey]
+	if ok {
+		memLimit, err := resource.ParseQuantity(memLimit)
+		if err != nil {
```

Comment:
this is repeatable code, can we extract out in helper , argument of that method would be resource key..

Reply:
I refactored the quantity parser in the last commit